data/resources.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "GDG DevFest Season 2017",
   "description": "The biggest Google tech conference in Kenya carefully crafted for you by the GDG community (GDG Kisumu, GDG Rongo, GDG Kisii and GDG MMUST)! All about Android, Web and Cloud from the world experts",
-  "keywords": "event, gdg, gde, devfest, google, programming, android, kisumu,  chrome, polymer, developers, web, cloud, androiddev",
+  "keywords": "event, gdg, gde, devfest, google, programming, android, kisumu,  chrome, polymer, developers, web, cloud, androiddev , iot",
```

Comment:
How did we forget about IOT, ongeza hadi arduino 😛

Reply:
Let me do so just now...